app/install/kubernetes.md

```diff
@@ -14,17 +14,24 @@ links:
   gh_repo: "https://github.com/Kong/kong-dist-kubernetes/"
 ---
 
-# Kubernetes Ingress Controller
+# Kubernetes Ingress Controller for Kong
 
-You can install Kong CE or EE using our official <a href="https://github.com/Kong/kubernetes-ingress-controller">Kubernetes Ingress Controller</a>.
+Install Kong Community Edition (CE) or Kong Enterprise Edition (EE) using the official 
+<a href="https://github.com/Kong/kubernetes-ingress-controller"> Kubernetes Ingress Controller</a>.
```

Comment:
Could we strip the whitespace within the link's anchor?

Reply:
Sorry, I'm not clear what you mean.

Reply:
Oh, I get it - fixing now...